src/routers/components/newBook.jsx

```diff
@@ -1,56 +1,36 @@
 import { useState } from 'react';
+import { useDispatch } from 'react-redux';
 import BookList from './aticles';
+import { addBook } from '../../redux/books/bookSlice';
 
 const AddNewBook = () => {
-  const [book, setBook] = useState([
-    {
-      id: Math.floor(Math.random() * (9999 - 1000)),
-      title: '',
-      author: '',
-    },
-    {
-      id: Math.floor(Math.random() * (9999 - 1000)),
-      title: '',
-      author: '',
-    },
-  ]);
+  const dispatch = useDispatch();
 
   const [title, setTitle] = useState('');
   const [author, setAuthor] = useState('');
 
-  const handleChangeTitle = (e) => {
-    setTitle(e.target.value);
-  };
-
-  const handleChangeAuthor = (e) => {
-    setAuthor(e.target.value);
-  };
-
   const handleSubmit = (e) => {
     e.preventDefault();
 
     const newBook = {
-      id: Math.floor(Math.random() * (9999 - 1000)),
       title,
       author,
+      category: '',
     };
-    setBook([...book, newBook]);
-  };
 
-  const delBook = (id) => {
-    setBook([
-      ...book.filter((book) => book.id !== id),
-    ]);
+    dispatch(addBook(newBook));
+    setTitle('');
+    setAuthor('');
   };
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍